acra-core/src/main/java/org/acra/sender/SenderService.java

```diff
@@ -99,13 +100,32 @@ protected void onHandleWork(@NonNull Intent intent) {
     @NonNull
     private List<ReportSender> getSenderInstances(@NonNull CoreConfiguration config) {
         List<Class<? extends ReportSenderFactory>> factoryClasses = config.reportSenderFactoryClasses();
-        List<ReportSenderFactory> factories = !factoryClasses.isEmpty() ? new InstanceCreator().create(factoryClasses) : config.pluginLoader()
-                .loadEnabled(config, ReportSenderFactory.class);
+
+        if (ACRA.DEV_LOGGING) ACRA.log.d(LOG_TAG, "config#reportSenderFactoryClasses : " + factoryClasses);
+
+        final List<ReportSenderFactory> factories;
+        if (factoryClasses.isEmpty()) {
+            if (ACRA.DEV_LOGGING) ACRA.log.d(LOG_TAG, "No reportSenderFactoryClasses - loading using PluginLoader to find ReportSenders");
```

Comment:
This sounds like a warning. Using the plugin loader is the default, I wouldn't warn if it is used

Reply:
OK, how about  "Using PluginLoader to find ReportSenderfactories"

Reply:
Sounds good